code/modules/unit_tests/rcd.dm

```diff
@@ -0,0 +1,23 @@
+/datum/unit_test/rcd/Run()
```

Comment:
Can you document what this is trying to test? It's not obvious through the code to me.

Reply:
Unit test moved to https://github.com/tgstation/tgstation/pull/56287